## From Debian's kqemu-source package:
## 03-2.5.22.dpatch by Samuel Tardieu <sam@rfc1149.net>
##
## DP: Fix FTBFS with linux 2.6.22 (Closes: #434031).

--- kqemu-linux.c.orig        2007-02-06 21:02:00.000000000 +0000
+++ kqemu-linux.c     2007-07-24 05:41:14.000000000 +0000
@@ -26,6 +26,7 @@
 #include <linux/ioctl.h>
 #include <linux/smp_lock.h>
 #include <linux/miscdevice.h>
+#include <linux/sched.h>
 #include <asm/atomic.h>
 #include <asm/processor.h>
 #include <asm/uaccess.h>
